README.md

```diff
@@ -162,18 +162,18 @@ For the convenience of the developer, we here provide the setup script which wil
   **usage:**
 
   ``` 
-  python3 MIVisionX-setup.py --directory [setup directory - optional (default:~/)]
-                             --opencv    [OpenCV Version - optional (default:3.4.0)]
-                             --miopen    [MIOpen Version - optional (default:2.11.0)]
-                             --miopengemm[MIOpenGEMM Version - optional (default:1.1.5)]
-                             --protobuf  [ProtoBuf Version - optional (default:3.12.0)]
-                             --rpp       [RPP Version - optional (default:0.7)]
-                             --ffmpeg    [FFMPEG Installation - optional (default:no) [options:yes/no]]
-                             --rocal     [MIVisionX rocAL Dependency Install - optional (default:yes) [options:yes/no]]
-                             --neural_net[MIVisionX Neural Net Dependency Install - optional (default:yes) [options:yes/no]]
-                             --reinstall [Remove previous setup and reinstall (default:no)[options:yes/no]]
-                             --backend   [MIVisionX Dependency Backend - optional (default:OCL) [options:OCL/HIP]]
-                             --rocm_path [ROCm Installation Path - optional (default:/opt/rocm) - ROCm Installation Required]
+  python MIVisionX-setup.py --directory [setup directory - optional (default:~/)]
```

Comment:
@kiritigowda Can be python or python3. We made that change a long time back along with model compiler. Please mention both

Reply:
Need a newer commit to support python3 for all OS options